Formula/ec2-api-tools.rb

```diff
@@ -18,7 +18,7 @@ class Ec2ApiTools < Formula
   # Deprecated upstream somewhere between 2017-12-24 and 2018-09-09 here:
   # https://docs.aws.amazon.com/AWSEC2/latest/APIReference/Welcome.html
   # The current version (1.7.5.1) is from 2015-09-08.
-  deprecate! date: "2020-02-03", because: :deprecated_upstream
+  disable! date: "2022-06-17", because: :deprecated_upstream
```

Comment:
May be better to simultaneously disable with `aws-cfn-tools` and `elb-tools`. I deprecated those recently (#98492) so not sure if we want to wait a bit longer or just rely on original `ec2-api-tools` deprecation date.